src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  const calculator = {
+    add(res = 0, num) {
+      result += num;
+
+      return calculator;
+    },
+    subtract(res = 0, num) {
+      result -= num;
+
+      return calculator;
+    },
+    multiply(res = 0, num) {
+      result *= num;
+
+      return calculator;
+    },
+    divide(res = 0, num) {
+      result /= num;
+
+      return calculator;
+    },
+    reset() {
+      result = 0;
+
+      return calculator;
+    },
+    get result() {
+      return result;
+    },
```

Comment:
this getter is redundant, result is supposed  be a property of calculator object